projects/Mallard/src/helpers/weather-hider.ts

```diff
@@ -3,19 +3,30 @@ import { largeDeviceMemory } from 'src/hooks/use-config-provider'
 import { setIsWeatherShown } from './settings/setters'
 import AsyncStorage from '@react-native-community/async-storage'
 import { errorService } from 'src/services/errors'
+import { getDefaultEditionSlug } from 'src/hooks/use-edition-provider'
 
 // Purpose: To hide the weahter on the first load unless the user turns it on
 // Intended for use on lower powered devices
 
 const KEY = '@weatherLowRAMCheck'
+const EDITIONCHECKKEY = '@weatherEditionCheck'
 
 const weatherHider = async (client: ApolloClient<object>) => {
     try {
         const weatherLowRamCheck = await AsyncStorage.getItem(KEY)
+        const editionWeatherCheck = await AsyncStorage.getItem(EDITIONCHECKKEY)
```

Comment:
Would recommend refactoring out the `AsyncStorage` here to use our helper in `storage.ts` to ensure consistency. Not required for this PR though.